themes/tdr/login/messages/messages_en.properties

```diff
@@ -20,7 +20,10 @@ loginTotpUnableToScan=I want to enter a key instead
 mainHeader=Transfer Digital Records
 passwordNew=New password
 passwordReset=Password reset
-passwordRestrictions=Password must be at least 10 characters
+passwordRestrictions=Password must be at least 8 characters
+invalidPasswordMinLengthMessage=Invalid password: Your password must contain a minimum of {0} characters
+invalidPasswordMinUpperCaseCharsMessage=Invalid password: Your password must include at least {0} uppercase character
```

Comment:
Should "character" be plural, ie "characters", or "character(s)"?

Reply:
The final design has it as "uppercase character" and "minimum of 8 characters"

Reply:
OK. Fair enough